app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,43 @@
+class PostsController < ApplicationController
+
+	def index
+		@posts = Post.all
+	end
+
+	def show
+		@post = Post.find(params[:id])
+	end
+
+	def new
+		@post = Post.new
+	end
+
+	def create
+		@post = Post.new(post_params)
+		@post.save
+		redirect_to @post
+	end
```

Comment:
You could also have added in some better error handling incase the create or update didn't go through:
https://github.com/ga-wdi-lessons/rails-errors-validations-flash/blob/master/validations.md#putting-it-all-together